libs/platform/kernel/ebpf_handle_kernel.c

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation
 // SPDX-License-Identifier: MIT
 
+#include "framework.h"
```

Comment:
Does this really need to be out of alphabetical order?  Can this move down under line 6?

Reply:
Unfortunately, it does due to it pulling in Windows headers that must be included first.